<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>795</width>
    <height>441</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="Op1label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Описание комнаты</string>
    </property>
   </widget>
   <widget class="QLabel" name="Op2label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>531</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Здесь будет описание текущей комнаты</string>
    </property>
   </widget>
   <widget class="QPushButton" name="NorthButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>280</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Север</string>
    </property>
   </widget>
   <widget class="QPushButton" name="WestButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>310</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Запад</string>
    </property>
   </widget>
   <widget class="QPushButton" name="EastButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>310</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Восток</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SouthButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>340</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Юг</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelInfo">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>10</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Информация о герое</string>
    </property>
   </widget>
   <widget class="QLabel" name="Heroname">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>40</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Имя героя</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelHealth">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>60</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Уровень здоровья</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="Health_of_hero">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>90</y>
      <width>191</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>100</number>
    </property>
   </widget>
   <widget class="QLabel" name="Money">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>120</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>50 монет</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelItems">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>150</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Список предметов</string>
    </property>
   </widget>
   <widget class="QListView" name="listView">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>180</y>
      <width>201</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="ShopButton">
    <property name="geometry">
     <rect>
      <x>584</x>
      <y>290</y>
      <width>191</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Купить что-нибудь</string>
    </property>
   </widget>
   <widget class="QLabel" name="Oplabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>521</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Здесь будет название текущей комнаты</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelGo">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>260</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Куда идти?</string>
    </property>
   </widget>
   <widget class="QPushButton" name="TakeMoney">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>584</x>
      <y>320</y>
      <width>191</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Получить монеты</string>
    </property>
   </widget>
   <widget class="QPushButton" name="suicide">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>584</x>
      <y>350</y>
      <width>191</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Понизить здоровье</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelItemsInCurrentRoom">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Предметы в комнате</string>
    </property>
   </widget>
   <widget class="QListView" name="listView_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>201</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>795</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
